app/models/server/models/LivechatInquiry.js

```diff
@@ -128,6 +128,23 @@ export class LivechatInquiry extends Base {
 		return this.findOne(query);
 	}
 
+	async getDistinctQueuedDepartments() {
```

Comment:
The initial implementation was using `distinct`, but since `undefined` value is ignored when passing any filter(such as `{ status: 'queued' }`) then I opted for an aggregation.
I implemented the `distinct` method again and now I'm adding `undefined` as default.
`Undefined`= public queue(when the inquiry is not associated with any department)